HISTORY.md

```diff
@@ -13,6 +13,8 @@
 
   * Fix Windows Python build configuration (#1885).
 
+  * Fix failure due to too large rho value (#2086).
```

Comment:
Could we be a little more descriptive in this changelog commit please?  I don't think most users will understand what this means.  I might suggest:

```
 * Correctly handle RNN sequences that are shorter than the value of rho (#2086).
```

Or something like that. :+1: